### PR TITLE
Graph plots

### LINKED ISSUE #15: Visualisation of Protein Graphs
- [x] 3d Interactive Viewer (using coords) - plotly
- [x] 3d static plot (using coords) - Nx & Matplotlib
- [x] 2d projection

Colouring options
- [x] Resiude type
- [x] Atom type
- [x] Edge types
- [x] Degree
- [x] Sequence position

Node Size options
- [x] Degree
- [ ] By feature magnitude (single or several?)

Labelling options
- [x]  Residue Name
- [x] Sequence Position
- [x] Identifier

Documentation
- [x] Docstrings
- [x] Docs

### PR DESCRIPTION
**Pull request by kamurani**
Fixes #15 

Add capability to use `rsa` and `dim_[1-7]` of the meiler embeddings as features by which the node size is scaled in `protein.visualisation.plotly_protein_structure_graph`.  

Testing by plotting graph:
rsa-node-size

### CODE REVIEW

File: graphein/protein/visualisation.py [hunk 1]
```diff
@@ -239,6 +243,21 @@ def plotly_protein_structure_graph(
...
 
+    # Get node size 
+    def node_scale_by(G, feature):
+        if feature == 'degree':
+            return lambda k : node_size_min + node_size_multiplier * G.degree[k]
+        elif feature == 'rsa':
+            return lambda k : node_size_min + node_size_multiplier * G.nodes(data=True)[k]['rsa']
+        # Meiler embedding dimension
+        p = re.compile("meiler-([1-7])")
+        if dim := p.search(feature).group(1):
```
Comment (a-r-j) [inline graphein/protein/visualisation.py:254]: Could you remove the walrus operator here? We're trying to maintain support for python 3.7 so Graphein can be used in Colab.

**Review by a-r-j — APPROVED** [review]
LGTM but for a minor nit. Thanks for the contribution!

File: graphein/protein/visualisation.py (hunk 1 above)
Comment (cimranm) [reply graphein/protein/visualisation.py]: @a-r-j I've taken it out, how's that?

**Comment by a-r-j** [comment]
LGTM, happy to merge. Could you quickly pull this up to head to incorporate the latest changes so tests can run?

**Comment by cimranm** [comment]
@a-r-j  sure thing; although I'm not exactly sure how I would go about doing that.  This is the first time i've submitted a PR sorry! 

Do you want me to re-commit to the already existing pull request? Or is there a simpler way? 

Cheers

**Pull request merged**